tests/expectations/inner_no_export.compat.c

```diff
@@ -0,0 +1,20 @@
+#include <stdint.h>
+#include <stddef.h>
+#include <stdbool.h>
+
+/**
+ * Outer docs should be exported.
+ */
+typedef struct {
+  Inner *inner;
```

Comment:
@emilio weird. When I use the same code for opendal c binding, it generates something like `struct Inner *inner` for such field. But in cbindgen tests it cannot generate the correct result.